elide-core/src/main/java/com/yahoo/elide/core/filter/Operator.java

```diff
@@ -536,6 +565,60 @@ private static <T> Predicate<T> hasMember(Path fieldPath, List<Object> values, R
         };
     }
 
+    private static <T> Predicate<T> subsetOf(Path fieldPath, List<Object> values, RequestScope requestScope) {
+        return (T entity) -> {
+            Type<?> valueClass = fieldPath.lastElement().get().getFieldType();
+
+            Object leftHandSide = getFieldValue(entity, fieldPath, requestScope);
+
+            BiPredicate<Object, Object> predicate = (a, b) -> a.equals(b);
+
+            List<?> rightHandSide = values.stream().map(value -> CoerceUtil.coerce(value, valueClass)).toList();
+
+            if (leftHandSide instanceof Collection<?> collection && !valueClass.isAssignableFrom(COLLECTION_TYPE)) {
+                for (Object left : collection) {
+                    if (!rightHandSide.stream().anyMatch(object -> predicate.test(left, object))) {
+                        // An element on the leftHandSide is not present on the rightHandSide
+                        return false;
+                    }
+                }
+                // Every element on leftHandSide is present on rightHandSide
+                return true;
+            } else {
+                // This happens when leftHandSide is null
+                // An empty set is a subset of every set
+                return true;
+            }
+        };
+    }
+
+    private static <T> Predicate<T> supersetOf(Path fieldPath, List<Object> values, RequestScope requestScope) {
+        return (T entity) -> {
+            Type<?> valueClass = fieldPath.lastElement().get().getFieldType();
+
+            Object leftHandSide = getFieldValue(entity, fieldPath, requestScope);
+
+            BiPredicate<Object, Object> predicate = (a, b) -> a.equals(b);
+
+            List<?> rightHandSide = values.stream().map(value -> CoerceUtil.coerce(value, valueClass)).toList();
+
+            if (leftHandSide instanceof Collection<?> collection && !valueClass.isAssignableFrom(COLLECTION_TYPE)) {
```

Comment:
Doesn't the case where leftHandSide is instanceof collection && valueClass **_is_** assignable to collection still need to be handled?